src/Heartbeat/Sensor/DBConnection.php

```diff
@@ -17,8 +17,8 @@ class DBConnection extends Sensor
     protected function _getStatus()
     {
         try {
-            ConnectionManager::get('default');
-            $hasDBConnection = true;
+            $connection = ConnectionManager::get('default');
+            return $connection->connect();
         } catch (\Exception $e) {
             $hasDBConnection = false;
```

Comment:
If we return early within the try block, we should return in the catch  block, too.